### PR TITLE
HPCC-24527 Document Single User Security Manager

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [x] Test my draft Pull Request.

## Testing:

successful completion of Unit tests: http://10.224.20.18/view/Docs-gp/job/DocBuild-05-GPanRepo04/

### CODE REVIEW

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml [hunk 1]
```diff
@@ -288,6 +288,26 @@
     </sect3>
   </sect2>
 
+  <sect2 id="SingleUserSecMgr">
+    <title>Single User Security Manager</title>
+
+    <para>The Single User security manager is a very specialized security
+    manager that allows a username/password combination to be specified on the
+    ESP startup command line. At run time, when you attempt to access any
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml:296]: run time s/b runtime

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml [hunk 2]
```diff
@@ -288,6 +288,26 @@
     </sect3>
   </sect2>
 
+  <sect2 id="SingleUserSecMgr">
+    <title>Single User Security Manager</title>
+
+    <para>The Single User security manager is a very specialized security
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml:294]: remove "very"

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

**Review by JamesDeFabia — APPROVED** [review]
Looks ok to me

**Review by ghalliday — CHANGES_REQUESTED** [review]
This contains unrelated changes to vcpkg and spark-hadoop

**Comment by g-pan** [comment]
Closing this PR with intent to reissue. Smoketest is reporting problems.

**Pull request closed without merging**